lightning/lightning.go

```diff
@@ -66,9 +67,15 @@ func New(globalCfg *config.GlobalConfig) *Lightning {
 		os.Exit(1)
 	}
 
+	tls, err := common.NewTLS(globalCfg.Security.CAPath, globalCfg.Security.CertPath, globalCfg.Security.KeyPath, globalCfg.App.StatusAddr)
+	if err != nil {
+		log.L().Fatal("failed to load TLS certificates", zap.Error(err))
```

Comment:
if `NewTLS` failed, should exit immediately？

Reply:
yes, for global config, currently it exits immediately